src/tests/ftest/io/nvme_io_verification.yaml

```diff
@@ -18,7 +18,7 @@ server_config:
  servers:
    bdev_class: nvme
    bdev_list: ["0000:5e:00.0","0000:5f:00.0"]
-   targets: 1
+   targets: 8
```

Comment:
The original intent of using `targets: 1` was to override the default (8).  Now that we will not need it, shouldn't we completely remove the `targets: <int>` entry, so the default is used?

This comment would apply to the other three yaml files in this PR

Reply:
done